### ISSUE TITLE
Player Data bug

### ISSUE THREAD

**Issue by blocker2508** [beauhobba/NRL-Data#1]
Had issues populating df in Player data model. .JSON has data from https://www.australiadatabases.com/sport

However when using this code to create NN model df doesn't populate correctly

**Comment by beauhobba** [comment]
Hi blocker2508, I am unsure of what could be causing this error, but I intend on doing an update in the coming weeks for the new season. As this model requires around 3 weeks of training data, I imagine I will update the scraper a week into the new season and then post an update.

**Comment by blocker2508** [comment]
look forward to the updates! happy to share my chunks that take the raw scores and uses softmax function to turn into odds.
